python/tests/test_tree_stats.py

```diff
@@ -2894,7 +2893,7 @@ def naive_branch_allele_frequency_spectrum(
     for j in range(num_windows):
         begin = windows[j]
         end = windows[j + 1]
-        for set_index, sample_set in enumerate(sample_sets):
+        for _set_index, _sample_set in enumerate(sample_sets):
             S = np.zeros(out_dim)
```

Comment:
Not sure what's happening here, but looks like an error.

Reply:
yep - holdover from a previous version

Reply:
Fixed